SU-SOE/stanford_soe_helper
# 7.x 2.x people spotlight

# READY FOR REVIEW

# Summary
Adding People spotlight

# Needed By (Date)
- ASAPP

# Urgency
Yes.

# Steps to Test

1. Pull this branch onto your site
2. Flush cache
3. Revert Feature
4. Verify:
- People Spotlight content type
- People Spotlight node display
- People Spotlight views
- People Spotlight manage content

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1937
- https://stanfordits.atlassian.net/browse/SOE-1938
- @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
